documentation/Using-the-SDK/Creating-your-plugin.md

```diff
@@ -1,27 +1,28 @@
 # Creating an SDK Plugin
 
 This document outlines how to use the Performance Toolkit SDK (SDK) to create
-an SDK plugin. A plugin can be used for processing trace files to be used by
-automation, trace extractors, or viewers such as Windows Performance Analyzer.
+an SDK plugin. Plugins can be used to expose data from `DataSource`s for
+automation, trace extractors, or viewers (e.g. Windows Performance Analyzer).
 
 Before creating a plugin, it is recommended to read [the overview of the SDK's architecture](../Architecture/Overview.md).
 
 Creating a plugin can be outlined into 4 distinct steps:
-* [Creating the Project](#creating-the-project)
-  * [Requirements](#requirements)
-  * [Creating Your Project](#creating-your-project)
-  * [Configuring Your Project](#configuring-your-project)
-    * [Add the Microsoft.Performance.SDK NuGet Package](#add-the-microsoftperformancesdk-nuget-package)
-    * [Picking your SDK version](#picking-your-sdk-version)
-    * [Install WPA for Debugging](#install-wpa-for-debugging)
-    * [Setup for Debugging Using WPA](#setup-for-debugging-using-wpa)
-* [Creating a ProcessingSource](#creating-a-processingsource)
-  * [Create a ProcessingSource class](#create-a-processingsource-class)
-  * [Decorate your ProcessingSource with the ProcessingSourceAttribute](#decorate-your-processingsource-with-the-processingsourceattribute)
-  * [Decorate your ProcessingSource with a DataSourceAttribute](#decorate-your-processingsource-with-a-datasourceattribute)
-  * [Implement the required ProcessingSource methods](#implement-the-required-processingsource-methods)
-* [Choosing a Plugin Framework](#choosing-a-plugin-framework)
-* [Creating a CustomDataProcessor and Tables](#creating-a-customdataprocessor-and-tables)
+1. [Creating the Project](#creating-the-project)
+    - [Requirements](#requirements)
+    - [Creating Your Project](#creating-your-project)
+    - [Configuring Your Project](#configuring-your-project)
+        * [Add the Microsoft.Performance.SDK NuGet Package](#add-the-microsoftperformancesdk-nuget-package)
+        * [Picking your SDK version](#picking-your-sdk-version)
+        * [Install WPA for Debugging](#install-wpa-for-debugging)
+        * [Setup for Debugging Using WPA](#setup-for-debugging-using-wpa)
+2. [Creating a ProcessingSource](#creating-a-processingsource)
```

Comment:
Looking at this list, maybe we should have all headers be in the gerund form. E.g. change "Decorate your.." to "Decorating your...". Thoughts?

Reply:
works for me!